server/flaskr/database/postgres.py

```
@@ -0,0 +1,51 @@
+        params = ("test", "Test")
+        cur.execute(query, params)
+
+        cur.execute(
+            sql.SQL("CREATE SCHEMA IF NOT EXISTS {}; ").format(sql.Identifier(str(orgId)))
+        )
+        cur.execute(
+            sql.SQL("SET search_path TO {}, public; ").format(sql.Identifier(str(orgId)))
+        )
+
+        with current_app.open_resource(
+            os.path.join("database", "sql", f"schema.sql")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)